docs/installing.md

```diff
@@ -1,6 +1,244 @@
+
+Finally, you'll need to set up a [GitHub webhook](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/about-webhooks) for reviews repository.
+This should be a repository that you have write access to,
+where you expect most of the reviewer-author interaction to occur.
+For JOSS, this corresponds to the `openjournals/joss-reviews` GitHub repository.
+
+In this GitHub repository's settings,
+you can add a new webhook with the following configuration:
+
+- Set the `Payload` URL to the `/dispatch` hook for your Heroku application URL.
+  For example, https://neurolibre.herokuapp.com/dispatch
+- Set the `Content type` to `application/json`
+- Set the secret to a high-entropy, random string as detailed in the [GitHub docs](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/securing-your-webhooks#setting-your-secret-token)
+- Set the webhook to deliver `all events`
+
+### Modifying your site contents
+
+You can modify site content by updating files in the `app` and `docs` folders.
+For example, in `app/views/notifications` you can text for any emails that will be sent by your application.
+
+Note that files which end in `.html.erb` are treated as HTML files, and typical HTML formatting applies.
+You can set the HTML styling by modifying the Sass files for your application,
+located in `app/assets/stylesheets`.
+
+There are currently a few hard-coded variables in the application which you will also need to update.
+Note that these are mostly under `lib/tasks`.
+For example, in `stats.rake`, the reviewer sheet ID is hard-coded on line 37.
+You should update this to point to your own spreadsheet where you maintain a list of eligible reviewers.
+
+In the same folder, `utils.rake` is currently hard-coded to alternate assignments of editor-in-chief based on weeks.
+You should modify this to either set a single editor-in-chief,
+or design your own scheme of alternating between members of your editorial board.
+
+## Deploying your Whedon-API Application
+
+Whedon-API can also be deployed on Heroku.
+Note that &mdash; for full functionality &mdash; Whedon-API must be deployed on [Hobby dynos](https://devcenter.heroku.com/articles/dyno-types), rather than free dynos.
+Hobby dynos allow the Whedon-API application to run continuously, without falling asleep after 30 minutes of inactivity;
+this means that Whedon-API can respond to activity on GitHub at any time.
+Whedon-API specifically requires two Hobby dynos: one for the `web` service and one for the `worker` service.
+
+On the Whedon-API Heroku deployment, you'll need to provision several [add-ons](https://elements.heroku.com/addons).
+Specifically, you'll need:
+
+1. [Cloudinary add-on](https://elements.heroku.com/addons/cloudinary)
+1. [Postgres add-on](https://elements.heroku.com/addons/heroku-postgresql),
+1. [Scheduler add-on](https://devcenter.heroku.com/articles/scheduler)
+1. [Redis To Go add-on](https://elements.heroku.com/addons/redistogo)
```

Comment:
No need for Postgres, Whedon-API deployments don't use any database (@arfon, please confirm).

Reply:
Correct. No PostgreSQL for Whedon

Reply:
Fixed in a500273 !